test/cdo/format/format-language-spec.js

```diff
@@ -0,0 +1,95 @@
+define([
+  'ccc/cdo',
+  'ccc/def'
+], function(cdo, def) {
+
+  describe("cdo.format.language -", function() {
+
+    it("Passing no arguments should return the current format provider", function() {
+      var lfp = cdo.format.language();
+
+      expect(def.classOf(lfp)).toBe(cdo.format);
+      expect(lfp.languageCode).toBe('neutral');
+    });
+
+    it("When one argument is given", function() {
+      it("And it is a string", function() {
+        it("Should return the format provider correspondent to the specified language", function() {
+          var lfp = cdo.format.language('neutral');
+          expect(def.classOf(lfp)).toBe(cdo.format);
+          expect(lfp.number().mask()).toBe('#,0.##');
+
+          it("And should return 'undefined' when there is none.", function(){
+            expect(cdo.format.language('SomeLanguage')).toBe(undefined);
+          })
+        });
+      });
+
+      it("And its an object should configure multiple format providers and return the last one.", function() {
```

Comment:
David, I overlooked this return value. The order of traversal of Object properties is not defined by ECMAScript. It won't work predictably in every engine. So we better just return cdo.format in this case.